extensions/vertx/deployment/src/test/java/io/quarkus/vertx/mdc/VertxMDCDevModeTest.java

```diff
@@ -0,0 +1,67 @@
+package io.quarkus.vertx.mdc;
+
+import static io.quarkus.vertx.mdc.VerticleDeployer.REQUEST_ID_HEADER;
+import static io.quarkus.vertx.mdc.VerticleDeployer.VERTICLE_PORT;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import org.awaitility.Awaitility;
+import org.awaitility.core.ThrowingRunnable;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusDevModeTest;
+
+public class VertxMDCDevModeTest {
+    @RegisterExtension
+    static final QuarkusDevModeTest TEST = new QuarkusDevModeTest()
+            .withApplicationRoot(
+                    (jar) -> jar.addClasses(JavaArchive.class, VerticleDeployer.class, InMemoryLogHandler.class,
+                            InMemoryLogHandlerProducer.class)
+                            .add(new StringAsset("quarkus.log.file.enable=true\n" +
+                                    "quarkus.log.console.format=%d{HH:mm:ss} %-5p requestId=%X{requestId} [%c{2.}] (%t) %s%e%n\n"),
+                                    "application.properties"))
+            .setLogFileName("quarkus-mdc.log");
+
+    @Test
+    void mdcDevMode() {
+        Path logDirectory = Paths.get(".", "target");
```

Comment:
It's equivalent to simply `Paths.get("target")` which in its turn calls `Path.of("target")`. You don't have to change it though.